src/analysis/normed_space/linear_isometry.lean

```diff
@@ -233,7 +233,11 @@ def subtypeₗᵢ : p →ₗᵢ[R'] E := ⟨p.subtype, λ x, rfl⟩
 @[simp] lemma subtypeₗᵢ_to_linear_map : p.subtypeₗᵢ.to_linear_map = p.subtype := rfl
 
 /-- `submodule.subtype` as a `continuous_linear_map`. -/
-def subtypeL : p →L[R'] E := p.subtypeₗᵢ.to_continuous_linear_map
+def subtypeL  {E : Type*} [add_comm_monoid E] [topological_space E] [module R' E]
+  (p : submodule R' E) : p →L[R'] E :=
+{ to_fun := λ f, f,
+  map_add' := λ f g, rfl,
+  map_smul' := λ c f, rfl, }
```

Comment:
Can you move this out of the `normed_space` folder and closer to `continuous_linear_map`?